pkg/identityserver/store/gateway.go

```diff
@@ -52,8 +52,9 @@ type Gateway struct {
 	StatusPublic   bool `gorm:"not null"`
 	LocationPublic bool `gorm:"not null"`
 
-	ScheduleDownlinkLate   bool `gorm:"not null"`
-	EnforceDutyCycle       bool `gorm:"not null"`
+	ScheduleDownlinkLate   bool   `gorm:"not null"`
+	EnforceDutyCycle       bool   `gorm:"not null"`
+	ScheduleAnytimeDelay   uint32 `gorm:"default:0"`
```

Comment:
@htdvisser what do we do with durations?

Reply:
As a first solution I have added the duration as an `int64` on the database, doing conversions on load/save.

gorm does not support time.Duration